style.css

```diff
@@ -1,12 +1,91 @@
+body{
+font-size: 25px;
+font-family: 'Smooch Sans', sans-serif;
+}
+header{
+  background-color: rgb(61, 61, 61);
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px;
+}
+
+header nav ul {
+  display: flex;
+  list-style: none;
+}
 
-.h1_title{
-  color: rgba(108, 137, 239, 0.909);
-  text-transform: uppercase;
+header nav ul li{
+position: relative;
+margin-right: 15px;
+}
+
+header nav ul li a{
+color: white;
+text-decoration: none;
+display: block;
+padding: 10px;
+}
+
+header nav ul li a:hover{
+  color: red;
+}
+
+.home-button:hover{
+text-decoration: underline;
+color: red;
+}
+
+.about-button:hover{
   text-decoration: underline;
+  color: red;
 }
-p{
-  color: darkolivegreen;
+
+.dropdown1{
+  position: relative;
 }
-.intro{
-  color: darksalmon;
+
+.services-dropdown{
```

Comment:
YOu have almost the same styles in "services-dropdown" and "products-dropdown". So you can use a common class "dropdown" for common styles and specific classes "services" and "products" for unique styles